x-pack/plugins/infra/public/utils/remote_state/remote_graphql_state.ts

```diff
@@ -179,9 +194,18 @@ export const createGraphqlStateSelectors = <State>(
     selectLoadingResult,
     result => (!isUninitializedLoadingResult(result) ? result.time : null)
   );
-  const selectIsUninitialized = createSelector(selectLoadingResult, isUninitializedLoadingResult);
-  const selectIsSuccess = createSelector(selectLoadingResult, isSuccessLoadingResult);
-  const selectIsFailure = createSelector(selectLoadingResult, isFailureLoadingResult);
+  const selectIsUninitialized = createSelector(
+    selectLoadingResult,
+    isUninitializedLoadingResult
+  );
+  const selectIsSuccess = createSelector(
+    selectLoadingResult,
+    isSuccessLoadingResult
+  );
+  const selectIsFailure = createSelector(
+    selectLoadingResult,
+    isFailureLoadingResult
+  );
```

Comment:
Kinda surprised prettier changed these lines, are you sure that a default option didn't change in prettier or one of it's plugins to make this happen? It seems like the line length requirement may have changed?

Reply:
it should have been caused by the yarn.lock upgrade from prettier 1.13 to prettier 1.15